### PR TITLE
Add methods for the organization model

### PR DESCRIPTION
**Pull request by thgreasi**
Change-type: minor
Signed-off-by: Thodoris Greasidis <thodoris@balena.io>

Resolves: # 
HQ: <url> 
See: <url> 
Depends-on: <url> 
Change-type: major|minor|patch 

---
##### Contributor checklist

- [ ] Includes tests
- [ ] Includes typings
- [ ] Includes updated documentation
- [ ] Includes updated build output

### CODE REVIEW

File: lib/models/organization.ts [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+		get(handleOrId, { $select: 'id' }).then(({ id }) => id);
+
+	/**
+	 * @summary Creates a new organization
+	 * @name create
+	 * @public
+	 * @function
+	 * @memberof balena.models.organization
+	 *
+	 * @description This method registers a new api key for the current user with the name given.
```
Comment (sradevski) [inline lib/models/organization.ts]: copy-paste 🚓

File: tests/integration/models/organization.spec.ts [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+				const orgs = await balena.models.organization.getAll();
+				expect(orgs).to.be.an('array');
+				expect(orgs).to.have.lengthOf(1);
+				expect(orgs[0]).to.have.property('handle', credentials.username);
+				ctx.userInitialOrg = orgs[0];
+			});
+		});
+
+		describe('balena.models.organization.get()', function() {
+			it('should be rejected if the application name does not exist', function() {
```
Comment (sradevski) [inline tests/integration/models/organization.spec.ts]: copy-paste 🚓

File: tests/integration/models/organization.spec.ts [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+				const randomTestOrgHandle = `testOrgRandom_${Date.now()}`;
+				const promise = balena.models.organization.get(randomTestOrgHandle);
+				return expect(promise).to.be.rejectedWith(
+					`Organization not found: ${randomTestOrgHandle}`,
+				);
+			});
+
+			it('should be rejected if the application id does not exist', function() {
```
Comment (sradevski) [inline tests/integration/models/organization.spec.ts]: copy-paste 🚓

File: tests/integration/models/organization.spec.ts [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+		describe('balena.models.organization.remove()', function() {
+			it('should be rejected if the organization handle does not exist', function() {
+				const randomTestOrgHandle = `testOrgRandom_${Date.now()}`;
+				const promise = balena.models.organization.get(randomTestOrgHandle);
+				return expect(promise).to.be.rejectedWith(
+					`Organization not found: ${randomTestOrgHandle}`,
+				);
+			});
+
+			it('should be rejected if the application id does not exist', function() {
```
Comment (sradevski) [inline tests/integration/models/organization.spec.ts]: copy-paste 🚓

**Comment by sradevski** [comment]
Also dont forget to generate the docs

**Comment by thgreasi** [comment]
So many missed copy pastes...

> Also dont forget to generate the docs

Will do it when merging the orgs branch to master.

**Pull request merged**